spack/package.py

```diff
@@ -71,6 +71,6 @@ def cmake_args(self):
             args.append('-DARB_GPU=cuda')
 
         # rely on spack's compiler wrapper to set architecture
-        args.append('-DARB_ARCH=')
-
-        return args
+        args.append('-DARB_ARCH=none')
+        opt_flags = spec.target.optimization_flags(spec.compiler.name, spec.compiler.version)
+        args.append('-DARB_CXX_FLAGS_TARGET="'+opt_flags+'"')
```

Comment:
using an f-string makes it easier to read imho

f'-DARB_CXX_FLAGS_TARGET="{opt_flags}"'

Reply:
I agree! However, don't we have to retain compatibility with older versions of Python for the spack package files? Would an f string be a problem there?

Reply:
You are right. Spack packages need to be Python 2.6 compliant.